brayns/common/types.h

```diff
@@ -335,6 +340,9 @@ enum class CameraMode
     inspect
 };
 
+// A clip plane is defined by a normal and a distance to the center of the scene bounding box
+typedef std::vector< Vector4f > ClipPlanes;
```

Comment:
Is it difficult to define clipping planes like the usual plane equation?
That's what RTNeuron uses and I think that Livre also.
I don't like your definition because if I have some clip planes and then I change the scene that would change the clip plane position, that makes no sense to me.